### ISSUE TITLE
"incomingStream" from other side fails upon this scenario [0.6.12]

### ISSUE THREAD

**Issue by amangupta26** [Temasys/SkylinkJS#227]
Using non-MCU App Key. Two Users A and B. Both join the room as follows:

```
function initSkyLink(){
    skylink.init({
      appKey: apikey,
      defaultRoom: CurrentRoom
    }, function(initError, initSuccess){
      if (initError)
        return console.log("initError", initError);

      if(group_id == 1 && isMCU)
        skylink._isPrivileged = true;
        
      skylink.joinRoom({
        audio: false,
        video: false,
        userData : {'name' : first_name}
      }, function(jRError, jRSuccess){
        if(jRError) return console.log("jRError", jRError);
        $('#enable').addClass('show');
      });
      
    });
  }
```
This function is called as soon as users come to website.
 
$('#enable') is the id of a button, which on clicking calls this function,
```
function sendMedia(){
  skylink.sendStream({
    audio : true,
    video : true
  });
}
```

Working scenario :
  A has enabled video and audio by clicking on "enable" button. Now B joins and clicks "enable" lazily i.e. not instantly. Then B receives A's stream successfully and vice-versa.

Not working scenario : 
  A has enabled video and audio by clicking on "enable" button. Now B joins and clicks "enable" instantly as soon as it appears(i.e. A has successfully joined the room). Then A **does not** receives B's stream . But A is able to receive B's stream.

Workaround: Add a random delay before calling skylink.sendStream(...), which I dont feel is good approach as may it still lead to failure and there is no deterministic delay too. 

Could help me to tackle this issue which pragmatically will happen rarely but may happen and is bad for user experience.

**Comment by letchoo** [comment]
Thanks @amangupta26 will take a look and notify you

**Comment by letchoo** [comment]
Hi @amangupta26,

We managed to reproduce the issue, and we will resolve that in the next release version.
For now, is it possible to use the timeout if possible.

The next release version would likely go out in the next few weeks.

**Comment by oooookk7** [comment]
Hi @amangupta26, please checkout [`0.6.15` release](https://github.com/Temasys/SkylinkJS/releases/tag/0.6.15) which should resolve your issue.

**Comment by oooookk7** [comment]
Closing this issue since it's been resolved in the `0.6.15` release. If this issue persists, feel free to re-open it again.
